my-code/movie/src/index.css

```diff
@@ -0,0 +1,244 @@
+:root {
+  /* Colors */
+  --yellow: #ff9f1c;
+  --red: #ff4040;
+  --green: #2ec4b6;
+  --dark: #0a1014;
+  --grey: #1b2329;
+  --mid-grey: #353f4c;
+  --light-grey: #7a8c99;
+  --white: #fff;
+}
+
+@import url('https://fonts.googleapis.com/css?family=Roboto:400,500,700');
+
+html, body {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+	height: 100%;
+	font: calc(100% + 0.1vw)/1.5 Roboto, -apple-system, system-ui, BlinkMacSystemFont, "Segoe UI";
```

Comment:
Cool! What exactly `calc(100% + 0.1vw)` means?

Reply:
The idea is that the font size becomes responsive. Inittially i had calc(100% + 0.25vw) but it was getting the font size too big. 100% beeing normally 16px, it grows when the screen gets wider. There's a way to truncate to a maximum with some more math also.